extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -28,9 +27,6 @@ export class AmpStoryRequestService {
 
     /** @private @const {!../../../src/service/xhr-impl.Xhr} */
     this.xhr_ = Services.xhrFor(win);
-
-    /** @const @type {function():(!Promise<!JsonObject>|!Promise<null>)} */
-    this.loadShareConfig = once(() => this.loadShareConfigImpl_());
```

Comment:
This was supposed to ensure we only load the sharing config once, even if called multiple times. Did you make sure that we're not trying to load the sharing config more than once?

Reply:
yes, i have migrated the call for the 'once' to amp-story-share.js - have tested in the browser and ran unit tests.